src/ContentBundle/Controller/Api/ContentController.php

```diff
@@ -420,4 +427,22 @@ public function duplicateAction(Request $request)
 
         return $response;
     }
+
+    /**
+     * @ApiDoc()
+     *
+     * @param Request $request
+     *
+     * @return JsonResponse
+     */
+    public function sitesAction(Request $request)
+    {
+        $sites = $this->get('opifer.cms.site_manager')->getSites();
```

Comment:
Could you please use `jms_serializer` here, instead of manually mapping the the collection to an array?

Something like the following;

```php
$sites = $this->get('opifer.cms.site_manager')->getRespository()->findAll();
$data = $this->get('jms_serializer')->serialize($sites, 'json');

return new JsonResponse(json_decode($data, true));
```